ladder/teams/README.md

```diff
@@ -0,0 +1,67 @@
+
+For example, the Cilium Project Code team, may require multiple GitHub teams in the Cilium GitHub organization to self-organize their work. 
+These multiple GitHub teams will be composed of contributors at different steps along the contributor ladder: organizational members, reviewers, and committers.  
+The different GitHub teams have different permissions, corresponding to the level of code access needed by the members of that team. 
+
+Cilium GitHub organization team memberships can be self-managed by making use of team membership configuration files in this directory.
+
+Each Cilium GitHub team can establish a file in this directory managed by the CODEOWNERS system with the assistance of 
+the Cilium GitHub organization [admins](https://github.com/cilium/community/blob/main/ADMINS-FOR-CILIUM.md) (or duly deputized members of the org tasked with helping initiating team self-management). 
+You can submit a PR against the Community repository to start the process to request a new team configuration and the correct Cilium contributors should be pulled into review.
+
+Changes to files in this directory, when merged into the main branch (after appropriate approvals and review), 
+will propogate via GitHub API automation and be reflected in the Cilium GitHub Organization's [teams](https://github.com/orgs/cilium/teams).
+
+## GitHub Team Member and Mentors
+With membership comes both priviledges and responsibilities.  One of the responsibilities of being on an active Cilium GitHub team member is 
+being available for automatic pull request review assignment for files owned by that GitHub team. 
+
+Over time as contributors transition from one GitHub team to another inside the project, while not active on a team any longer, 
+their historic experience may be needed to help review proposed changes with functionality they have insight into.  In these cases these 
+individuals can be added to the team's mentor list in the team configuration file here. Mentors are no longer part of the auto-assigned PR review rotation, 
+but they can still keep team approval access. Active Mentors of a team are expected to provide guidance to others in the team and seek reviews they 
+should participate in for the code owner area.
+
+## Don't Forget to Set Your GitHub Status
+When active team members are not available for an extended period of time (for example they are on vacation for several days), 
+they are encouraged to set themselves as busy in their [GitHub status](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#setting-a-status). If your status is busy GitHub will skip you to the pull request review rotation for the teams you are on.
```

Comment:
nit: It'd be nice to do a full scan of the text to write it in the way as though this document is talking directly to the reader. Example:

```suggestion
As an active team member, if you are not available for an extended period of time (for example, you go on vacation for several days), 
set yourself as busy in the [GitHub status](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#setting-a-status). If your status is busy GitHub will skip you to the pull request review rotation for the teams you are on.
```